src/remarx/app.py

```diff
@@ -0,0 +1,86 @@
+import marimo
+
+__generated_with = "0.14.16"
+app = marimo.App(width="medium")
+
+
+@app.cell
+def _():
+    import subprocess
+    import sys
+
+    import marimo as mo
+
+    import remarx
+    return mo, remarx, subprocess, sys
+
+
+@app.cell
+def _(mo, subprocess, sys):
+    # Launch notebook if its being run as as script
+    if not mo.running_in_notebook():
+        try:
+            subprocess.run(["marimo", "run", __file__])
```

Comment:
I  checked my local `remarx-app` script to make sure I was running the updated version, and it looked remarkably familiar.

The marimo command-line script is generated the same way, it's a package script that calls the cli main  method.

https://github.com/marimo-team/marimo/blob/main/pyproject.toml#L72-L73

Let's run it the same way instead of using subprocess.

The marimo CLI is built using click (which I know of but am not super familiar with) - I was looking briefly on stack overflow and just tested in a python console, you can call it directly with the args like this:
```python
main(['run', 'src/remarx/app.py'])
```

Reply:
Preferable to subprocess and fine for now. Did you see my note in an earlier review about opening an issue on the marimo repo to ask about an approved way to do this?